azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -798,6 +799,13 @@ private ImmutableMap getLabelsForPod(final int executionId) {
       mapBuilder.put("service", String.join("-", SERVICE_SELECTOR_PREFIX,
           clusterQualifiedExecId(this.clusterName, executionId)));
     }
+
+    // Set the label for disabling pod-cleanup.
+    if (flowParam != null && !flowParam.isEmpty() && flowParam
+        .containsKey(FlowParameters.FLOW_PARAM_DISABLE_POD_CLEANUP)) {
```

Comment:
Shouldn't you add value check as well? With that, shall it be admin only flow param?

Reply:
You're right, it should be an admin only param, adding that and a boolean value check as well.